kpi/serializers/v2/asset_export_settings.py

```diff
@@ -38,13 +39,15 @@ class Meta:
         fields = (
             'uid',
             'url',
+            'exports_url',
             'name',
             'date_modified',
             'export_settings',
         )
         read_only_fields = (
             'uid',
             'url',
+            'exports_url',
```

Comment:
FYI: `SerializerMethodField` is already read-only.  I think that `read_only_fields` is useless in that case.